### DISCUSSION TITLE
Failed to lookup docker package when repo. name is uppercase (but docker requires it to be lowercase)

### DISCUSSION THREAD

**Discussion by arodindev** [renovatebot/renovate#22323] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

Hello, 

i have a test-organization named ``10Testorga`` and i published an app under ``ghcr.io/10Testorga/flaskapp``. Docker requires the repository name to be lowercase, so i changed my docker and kubernetes config files to ``ghcr.io/10testorga/flaskapp``. 

However, renovate fails to look up the package ``ghcr.io/10testorga/flaskapp``. When i change it back to uppercase, then renovate works as expected but the image can not be pulled anymore (e.g. by kubernetes or github actions).

Is there any workaround to parse repo. name in the ``renovate.json``?

### What you tried so far.

Tried to parse the repo. name within ``renovate.json`` but failed to do so.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
You didn't say it explicitly, but it's a private image, right?

And the problem seems related to authentication? i.e. it it were public you wouldn't have the same problem?

We could either:
- Always do out matchHost case-insensitive, or
- massage the matchHost specifically for ghcr.io

> When i change it back to uppercase, then renovate works as expected

Assuming the image is private, do you mean that when the image matches your org name, Renovate an look up its tags without you needing to add in explicit authentication for it? (i.e. vanilla app config, no extra hostRules)

**Reply by arodindev** [reply]
No, it's a public image. I re-triggered the bot multiple times and also re-installed the app - nothing worked. However, after a few commits the problem has solved itself. Kinda weird, but ill take it.
